dali/test/python/test_operator_crop_mirror_normalize.py

```diff
@@ -0,0 +1,117 @@
+        self.decode = ops.HostDecoder(device = "cpu", output_type = types.RGB)
+        if self.is_new_cmn:
+            self.cmn = ops.NewCropMirrorNormalize(device = self.device,
+                                                  output_dtype = output_dtype,
+                                                  output_layout = output_layout,
+                                                  crop = (224, 224),
+                                                  crop_pos_x = 0.3,
+                                                  crop_pos_y = 0.2,
+                                                  image_type = types.RGB,
+                                                  mean = mean,
+                                                  std = std,
+                                                  pad_output = pad_output)
+        else:
+            self.cmn = ops.CropMirrorNormalize(device = self.device,
+                                               output_dtype = output_dtype,
+                                               output_layout = output_layout,
+                                               crop = (224, 224),
+                                               crop_pos_x = 0.3,
+                                               crop_pos_y = 0.2,
+                                               image_type = types.RGB,
+                                               mean = mean,
+                                               std = std,
+                                               pad_output = pad_output)
+        self.coin = ops.CoinFlip(probability=mirror_probability)
+
+    def define_graph(self):
+        inputs, labels = self.input(name="Reader")
+        images = self.decode(inputs)
+        if self.device == 'gpu':
+            images = images.gpu()
+        rng = self.coin()
+        images = self.cmn(images, mirror=rng)
+        return images
+
+def check_cmn_cpu_vs_gpu(batch_size, output_dtype, output_layout, mirror_probability, mean, std, pad_output):
+    compare_pipelines(CropMirrorNormalizePipeline('cpu', batch_size, output_dtype=output_dtype,
+                                                  output_layout=output_layout, mirror_probability=mirror_probability,
+                                                  mean=mean, std=std, pad_output=pad_output,
+                                                  is_new_cmn=False),
+                      CropMirrorNormalizePipeline('gpu', batch_size, output_dtype=output_dtype,
+                                                  output_layout=output_layout, mirror_probability=mirror_probability,
+                                                  mean=mean, std=std, pad_output=pad_output,
+                                                  is_new_cmn=False),
+                      batch_size=batch_size, N_iterations=10)
+
+def test_cmn_cpu_vs_gpu():
+    for batch_size in [1]:
+        for output_dtype in [types.FLOAT]:
```

Comment:
Int and half?

Reply:
int: done
half: requires adding support for float16